README.md

```diff
@@ -116,38 +116,38 @@ merged in close proximity. Our Concourse pipeline has jobs to automatically hand
 signing, and uploading production releases of the app.
 
 1. After you've merged all PRs to be included in the release, ensure the `.update-priority`
-file is set correctly according to the table below. Use priority 2 as the default. If you want to
-use priority 4 or 5, post in #apiary-mobile first.
+file is set correctly according to the table below. Use priority 1 as the default. If you want to
+use priority 5, post in #apiary-mobile first.
    1. Update priority affects if and how often users receive in-app update prompts to update the
 app to the latest version.
+2.
```

Comment:
Is this supposed to be here?

Reply:
Needed to update for the new release process, done